spec/trimmer_decorator_spec.rb

```diff
@@ -8,4 +10,10 @@
   it 'should return NotImplementedError' do
     expect { @trimmer_decorator.correct_name }.to raise_error(NotImplementedError)
   end
+
+  it 'should return the trim name' do
+  person = Person.new(25, 'John Doe')
```

Comment:
- [ ] [OPTIONAL] In order to write solid test cases, you should provide a better example name here which actually is more than 11 characters then you can check that trimmer decorator works as expected.